### ISSUE TITLE
Alt Text Missing from Media Settings

### ISSUE THREAD

**Issue by SiobhyB** [wordpress-mobile/WordPress-iOS#11008]
This is a subset of https://github.com/wordpress-mobile/WordPress-iOS/issues/7596. 

There isn't currently an option to add alt text to an image within the iOS app. This makes it harder for users to make their images accessible to people using screen readers.

This was reported to us by a blind blogger in 1773488-zen.

**Comment by SiobhyB** [comment]
Closing as it is actually possible to add alt text via the app (I should have double checked first, sorry!) It seems like the user's report is related to something else. I'll gather more details and start a new issue if necessary.
